### ISSUE TITLE
Module not working with Netlify

### ISSUE THREAD

**Issue by schneider-carlo** [harlan-zw/nuxt-og-image#52]
### Describe the bug

When using this module on netlify you will get a white box when you open up the url for the og:image instead of the correct og:image. 
Example links: 
`https://nuxt-og-image-issue.netlify.app` 
`https://nuxt-og-image-issue.netlify.app/__og_image__/og.png`

Reprodction: 

1. login to netlify and upload this git repository: git@github.com:schneider-carlo/nuxt-og-image-issue.git
2. set the correct base url as env variable with the name `BASE_URL`
3. when netlify has finished the building open up the new page
4. go to the source code and open the og:image link 
5. now you should also see the white box with the missing image

### Reproduction

git@github.com:schneider-carlo/nuxt-og-image-issue.git

### System / Nuxt Info

```Shell
- Operating System: `Darwin`
- Node Version:     `v18.16.0`
- Nuxt Version:     `3.5.1`
- Nitro Version:    `2.4.1`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `modules`, `ogImage`
- Runtime Modules:  `nuxt-og-image@2.0.0-beta.40`
- Build Modules:    `-`
```

**Comment by harlan-zw** [comment]
Can confirm that I can replicate this, thanks for tbe details.

**Comment by harlan-zw** [comment]
Thanks for the issue and reproduction @schneider-carlo! Managed to track down the issue to a Nitro bug and pushed up a fix in 2.0.0-beta.42

Can you try and let me know how you go?
